plugins/modules/rds_instance.py

```diff
@@ -99,6 +99,7 @@
         description:
           - The identifier of the CA certificate for the DB instance.
         type: str
+        version_added: 6.1.0
```

Comment:
I guess e do not need to add version_added here because the parameter was already in there.